src/app/dataModels/project.ts

```diff
@@ -12,4 +12,5 @@ export interface Project {
   location?: string;
   status?: ProjectStatusEnum;
   contributors?: string[];
+  className?: string;
```

Comment:
why are you including this on the project model?

Reply:
To override the width and height of the project card

Reply:
Addressed in [d7b07af](https://github.com/OpenBCca/openbc-web/pull/92/commits/d7b07af30c35ea8a196f645b9bedd6599ab3488c)